Library/Sender.ts

```diff
@@ -428,7 +429,7 @@ class Sender {
         try {
              //create file - file name for now is the timestamp, a better approach would be a UUID but that
             //would require an external dependency
-            var fileName = new Date().getTime() + ".ai.json";
+            var fileName = `${new Date().getTime()}.${this._processId}.ai.json`;
```

Comment:
How many digiths is the processId? I wonder if this could cause issues if the file name gets very long.

Reply:
We may just drop the date now right?, maybe use a UUID like the comment below?

Reply:
Looks like the max length for both Windows and Linux should be 255 characters for a file path. I'll add some tests for long file names, but assuming the above holds true. I'm comfortable using up a max of < 8% of our file name length for the PID.

Reply:
> We may just drop the date now right?, maybe use a UUID like the comment below?

Are you suggesting we just drop the PID & Date approach and transfer entirely to using the UUID package?

Reply:
Is relying on the external dependency no longer a concern?